### PR TITLE
fix component destroy leak

### LINKED ISSUE #6499: ngx-bootstrap ComponentLoader have to run destroy() method when component hide?
Bug description: Open and close typeahead dropdown multiple times lead to memory leaks (slow down browser). 
Open and close multiple times dropdown -slow down hovering over items (looks like they still exist in browser memory)
Same effect exist with other components that use ngx ComponentLoader

How to solve this issue in true way?

Plunker/StackBlitz that reproduces the issue:
StackBlitz: https://stackblitz.com/edit/angular-ivy-ez8g8e?file=src%2Fapp%2Fapp.module.ts

I try to provide same ComponentLoader, but with running method destroy() if component hide - 
271. this._componentRef.destroy(); 

sample of first open
first-open

sample after 10 times open
after-10-times-open

### PR DESCRIPTION
**Pull request by sherlock1982**
fixes #6499 #5999

### CODE REVIEW

**Review by Alexveselun — APPROVED** [review]
Testing completed. Locally work correctly. No critical issues were found.

**Comment by sherlock1982** [comment]
Are these failures caused by my changes or it's just something random? I checked one of the tests and it looks like it's just stuck in the middle.

**Comment by SvetlanaMuravlova** [comment]
> Are these failures caused by my changes or it's just something random? I checked one of the tests and it looks like it's just stuck in the middle.

failures in full e2e tests don't connect with your changes, not all tests are fixed, for some reasons

File: src/component-loader/component-loader.class.ts [hunk 1]
```diff
@@ -228,6 +228,7 @@ export class ComponentLoader<T extends object> {
     this._contentRef?.viewRef?.destroy();
 
     this._contentRef = void 0;
+    this._componentRef.destroy();
```
Comment (valorkin) [inline src/component-loader/component-loader.class.ts:232]: I would go with ` this._componentRef?.destroy()`

**Comment by lexasq** [comment]
Was merged another PR with the same code

**Pull request closed without merging**